wp-content/themes/eighshift-task/src/blocks/custom/posts/parts/post.js

```diff
@@ -0,0 +1,81 @@
+import classnames from 'classnames';
+
+export const Post = (props) => {
+  const {
+    parentClass,
+    title,
+    excerpt,
+    featuredImageThumb,
+    categories,
+    date,
+  } = props;
+
+  const postClasses = classnames([
+    `${parentClass}__post`,
+  ]);
+
+  const imgClasses = classnames([
+    `${parentClass}__post-image`,
+  ]);
+
+  const dateClasses = classnames([
+    `${parentClass}__post-date`,
+  ]);
+
+  const headingClasses = classnames([
+    `${parentClass}__post-title`,
+  ]);
+
+  const excerptClasses = classnames([
+    `${parentClass}__post-excerpt`,
+  ]);
+
+  const categoriesClasses = classnames([
+    `${parentClass}__post-category-list`,
+  ]);
+
+  const categoryClasses = classnames([
+    `${parentClass}__post-category`,
+  ]);
+
+  return (
+    <div className={postClasses}>
+      {
```

Comment:
please compress this as we agreed on php part 
`{featuredImageThumb &&
       <img src={featuredImageThumb[0]} alt="" className={imgClasses} />}`